### PR TITLE
feat(typings): add icon names for Icon

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/elements/Icon/Icon.d.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ export interface IconProps {
   loading?: boolean;
 
   /** Name of the icon. */
-  name?: string;
+  name?: SemanticICONNAMES;
```
Comment (layershifter) [inline src/elements/Icon/Icon.d.ts]: Can we rename it to `SemanticICONS`? It will have more parity with `SemanticCOLORS`, `SemanticWIDTHS`, etc.

**Review by layershifter — CHANGES_REQUESTED** [review]
@vincealdrin Thanks for PR, I left only one comment about naming.

In fact, this will not cover all possible values. Typings allow `mail outline` while `outline mail` is also valid. But, I'm sure that we don't need to cover these cases.

File: src/elements/Icon/Icon.d.ts (hunk 1 above)
Comment (ghost) [reply src/elements/Icon/Icon.d.ts]: yeah sure

**Review by layershifter — APPROVED** [review]

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.75.0`

**Pull request merged**